libs/cmdio/logger.go

```diff
@@ -104,6 +105,36 @@ func AskYesOrNo(ctx context.Context, question string) (bool, error) {
 	return false, nil
 }
 
+func AskSelect(ctx context.Context, question string, choices []string) (string, error) {
+	logger, ok := FromContext(ctx)
+	if !ok {
+		logger = Default()
+	}
+	return logger.AskSelect(question, choices)
+}
+
+func (l *Logger) AskSelect(question string, choices []string) (string, error) {
```

Comment:
We have a very similar code already here, can it be modified to reuse?

https://github.com/databricks/cli/blob/35e8ed30c6207c239331c30491ecd2c34b1d123e/libs/cmdio/io.go#L135

Reply:
IMO it's fine to have separate functions because all we do here is to call into promptui with a configuration that's different from the usecase that already exstings. 

We could use the configuration in the other function, but IMO the current experience is cleaner and more consistent with the custom prompting logic we have for non enum values.

FYI, Here's what a select looks like with the existing function:

<img width="432" alt="Screenshot 2023-08-16 at 5 48 26 PM" src="https://github.com/databricks/cli/assets/88374338/e6a8a0a9-ffb3-4d28-bcf9-39df7cfd0cd7">

Reply:
Created internal issue to track.